packer/config.plist

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>ConfigurationVersion</key>
+	<integer>2</integer>
+	<key>Debug</key>
+	<dict>
+		<key>DebugLog</key>
+		<false/>
+	</dict>
+	<key>Display</key>
+	<dict>
+		<key>ConsoleFont</key>
+		<string>Menlo</string>
+		<key>ConsoleFontSize</key>
+		<integer>12</integer>
+		<key>ConsoleTheme</key>
+		<string>Default</string>
+		<key>DisplayCard</key>
+		<string>virtio-ramfb</string>
+		<key>DisplayDownscaler</key>
+		<string>linear</string>
+		<key>DisplayFitScreen</key>
+		<true/>
+		<key>DisplayRetina</key>
+		<false/>
+		<key>DisplayUpscaler</key>
+		<string>linear</string>
+	</dict>
+	<key>Drives</key>
+	<array>
+		<dict>
+			<key>DriveName</key>
+			<string>drive0</string>
+			<key>ImagePath</key>
+			<string>image.qcow2</string>
+			<key>ImageType</key>
+			<string>disk</string>
+			<key>InterfaceType</key>
+			<string>virtio</string>
+		</dict>
+		<dict>
+			<key>DriveName</key>
+			<string>drive1</string>
+			<key>ImageType</key>
+			<string>cd</string>
+			<key>InterfaceType</key>
+			<string>usb</string>
+			<key>Removable</key>
+			<true/>
+		</dict>
```

Comment:
Does this still need to be present in the final `.plist`?

Reply:
Probably not. It shows up everywhere else, but isn't necessary in the final image here.